robotidy/transformers/RenameKeywords.py

```diff
@@ -125,16 +128,23 @@ def remove_underscores_and_capitalize(self, value: str):
         if self.remove_underscores:
             value = value.replace("_", " ")
             value = re.sub(r" +", " ", value)  # replace one or more spaces by one
-        words = []
-        split_words = value.split(" ")
-        # capitalize first letter of every word, leave rest untouched
-        for index, word in enumerate(split_words):
-            if not word:
-                if index in (0, len(split_words) - 1):  # leading and trailing whitespace
-                    words.append("")
-            else:
-                words.append(word[0].upper() + word[1:])
-        return " ".join(words)
+
+        if self.convert_title_case:
+            words = []
+            split_words = value.split(" ")
+
+            # capitalize first letter of every word, leave rest untouched
+            for index, word in enumerate(split_words):
+                if not word:
+                    if index in (0, len(split_words) - 1):  # leading and trailing whitespace
+                        words.append("")
+                else:
+                    words.append(word[0].upper() + word[1:])
+            result = " ".join(words)
+        else:
+            result = value
+
+        return result
```

Comment:
Condition can be reversed for readability:

```suggestion
        if not self.convert_title_case:
            return value
        words = []
        split_words = value.split(" ")

        # capitalize first letter of every word, leave rest untouched
        for index, word in enumerate(split_words):
            if not word:
                if index in (0, len(split_words) - 1):  # leading and trailing whitespace
                    words.append("")
            else:
                words.append(word[0].upper() + word[1:])
       return " ".join(words)
```